app/src/db/migrations/20240925201114_049-form-metadata.js

```diff
@@ -0,0 +1,27 @@
+const stamps = require('../stamps');
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+
+exports.up = function (knex) {
+  return Promise.resolve().then(() =>
+    knex.schema.createTable('form_metadata', (table) => {
+      table.uuid('id').primary();
+      table.uuid('formId').references('id').inTable('form').notNullable().index();
+      table.string('headerName', 255).notNullable();
+      table.string('attributeName', 255).notNullable();
```

Comment:
It seems like these aren't used by the frontend. How soon will the frontend be changed to allow the user to set them? We don't usually add things like this until they're needed, to keep the size of PRs down and so that they don't have to be removed down the line if setting them is never implemented.

Reply:
removed